### DISCUSSION TITLE
á = a

### DISCUSSION THREAD

**Discussion by 3tyh86qvt** [standardnotes/forum#2640] [Suggestions]
When using the integrated search box, "fábrica" and "fabrica" yield completely different results, although it's the same meaning with a simple stress variation. An accurate search implementation is thus creating an unnecessary complication.

Now, when I look up "fábrica", Standard Notes is not able to find "fabrica". That's not good design practice.
In German, as in Spanish, some third-person present declensions do change how vowels are written, because there's a shift: ("Ich Schlage" > "er schlägt"). In this case again, looking up "schlagt" should not discriminate "schlägt". Because, as you can see, it's the same root, the same meaning just a different writing because of the implementation of tildes which English, for better or worse, has not done.

Most Latin script languages (English, Spanish, German, etc.) which resort to tildes (Spanish, German, French, etc.) do not have more vowels than the standard 5 vowel system (a, e, i, o, u).
Spanish uses tildes: "á", but Spanish is not a 10-vowel system. Spanish has 5 vowels.

Whether a word carries a tilde relies on the word itself, not on the letter itself. Words have tildes, vowels don't have tildes. Tildes are sometimes used to make explicit how a word should be read or pronounced. It would be of help to use tildes in English, as well: "the cómbat", "he combáts". In Spanish: "la fábrica", "él fabrica" ([fabríka]), But, as you can see, "á" is still an "a" and should not be excluded from searches. Looking up "fabrica" should give you "fábrica" back too.

Therefore, when looking up for texts, "a" should include "á". Maybe not the other way around. "Á" is more exclusive, carries a little bit more information, therefore, "fábrica" could yield different results. It'd be pretty irrelevant at the end of the day.

Vowels don't have tildes, words do.

**Comment by JaspalSuri** [comment]
Hi @3tyh86qvt, thank you for the feedback! I'll take note of this suggestion for the dev team to review when they revisit how search works across the apps.
